tests/test_pages.py

```diff
@@ -41,7 +41,7 @@ def to_item(self) -> dict:  # type: ignore
                 "title": self.css("title::text").get("").strip(),
             }
 
-    page_object = MyWebPage(book_list_html_response)
+    page_object = MyWebPage(response=book_list_html_response)
```

Comment:
Making it kw_only makes total sense to me, we should have done it from the start.

Does it mean though that user-defined dependencies also need to be kw-only now? It could mean that we need to update a examples in the docs; it might also break code.

Reply:
I've got a way around it by using an `attrs.field` converter in ItemPage for SelectPage. In this way, we don't have to break any code for now. fb7b7bd